File: npm/examples/app-generation.ts
import { Buble } from '@buble/sdk';

const buble = new Buble();

const task = await buble.apps.generations.create('asmr-crushing-frozen-fruits', {
  fruit: 'Strawberries',
  video_ratio: '16:9',
  video_resolution: '720p'
});

const result = await buble.apps.generations.wait(
  'asmr-crushing-frozen-fruits',
  task.data.id
);

console.log(result.data.result?.videos?.[0]?.url);

File: flutter/example/app_generation.dart
import 'package:buble/buble.dart';

Future<void> main() async {
  final client = BubleClient.fromEnvironment();

  final task = await client.apps.generations.create(
    'video-background-remover',
    {
      'source_video': ['https://example.com/source.mp4'],
      'refine_foreground_edges': true,
      'subject_is_person': true,
    },
  );

  final result = await client.apps.generations.wait(
    'video-background-remover',
    task.data.id,
  );
  print(result.data.result?.videos.firstOrNull?.url);
}
